Dockerfile

```diff
@@ -0,0 +1,28 @@
+#	GCCVERSION=$($HOST-g++ -E -dM $(mktemp --suffix=.h) | grep __VERSION__ | cut -d ' ' -f 3 | cut -d '"' -f 2) && \
+# TODO: Make a nicer way to get gcc version. But only major and minor
```

Comment:
It not used now, put in an issue for future improvement instead of leaving behind in the Dockerfile